src/map/lua/luautils.h

```diff
@@ -167,6 +167,7 @@ namespace luautils
     int32 clearVarFromAll(lua_State *);                                         // Deletes a specific player variable from all players
     int32 terminate(lua_State*);                                                // Logs off all characters and terminates the server
 
+    int32 GetTextIDVariable(lua_State*);                                        // Lua binding to call luautils::GetTextIDVariable
```

Comment:
so it's a lua binding to call a c++ binding to call lua?

I can't think of a single use for this